ApiBundle/Controller/ContentController.php

```diff
@@ -259,4 +259,23 @@ protected function findOneContent($contentId, $language, $version = null)
 
         return $content;
     }
+
+    /**
+     * @param boolean|null $published
+     *
+     * @Config\Route("/list/not-published-by-author", name="open_orchestra_api_content_list_author_not_published", defaults={"published": null})
+     * @Config\Route("/list/by-author", name="open_orchestra_api_content_list_author", defaults={"published": false})
+     * @Config\Method({"GET"})
+     *
+     * @Config\Security("has_role('ROLE_ACCESS_CONTENT_TYPE_FOR_CONTENT')")
+     *
+     * @return FacadeInterface
+     */
+    public function listContentByAuthor($published)
+    {
+        $user = $this->get('security.token_storage')->getToken()->getUser();
+        $content = $this->get('open_orchestra_model.repository.content')->findByAuthor($user->getUsername(), $published, 10);
```

Comment:
The author is the person who created the content I think, not the one modifying it. So what the widget must show ? Our creations or our modifications ?